Dependencies/xr/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ else()
     endif()
 
     set(SOURCES ${SOURCES}
+        "Source/OpenXR/Include/IXrContextOpenXR.h"
+        "Source/OpenXR/Include/XrContextHelperOpenXR.h"
```

Comment:
This line should be excluded, right? This isn't for Babylon Native, it's for other consumers, unless I misunderstand its purpose.

Reply:
this is only added for easing the developer process. This file doesn't get included in any static libs, the helper function also isn't enabled by default in BabylonNative. it does however allow you to view and edit the file in the cmake generated visual studio solution

Reply:
moving to BabylonReactNative